Sources/RswiftCore/SwiftTypes/Module.swift

```diff
@@ -9,23 +9,23 @@
 
 import Foundation
 
-enum Module: ExpressibleByStringLiteral, CustomStringConvertible, Hashable {
+public enum Module: ExpressibleByStringLiteral, CustomStringConvertible, Hashable {
```

Comment:
The `rswift` executable now depends on the module `RswiftCore`. These need to be public to be used from the executable.